core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -168,7 +166,7 @@ var ExplorationEditorMainTab = function() {
     // Otherwise, if the editor tutorial shows up, exit it.
     var buttons = element.all(by.css('.skipBtn'));
     if (await buttons.count() === 1) {
-      await (await buttons.get(0)).click();
+      await action.click('Skip button', await buttons.get(0));
```

Comment:
don't need the `await` before `buttons.get(0)`

Reply:
done